### PR TITLE
Replace bucket ID by bucketName in submitFromCatalog method

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**
The test in Jenkins fails, because the catalog-client changes should be merged before

### CODE REVIEW

File: rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java [hunk 1]
```diff
@@ -223,9 +223,9 @@ public JobId submit(URL job)
     }
 
     @Override
-    public JobId submitFromCatalog(String catalogRestURL, String bucketId, String workflowName)
+    public JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```
Comment (sonarqube-activeeon) [inline rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

File: rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java [hunk 2]
```diff
@@ -247,10 +247,10 @@ public JobId submit(URL job, Map<String, String> variables)
     }
 
     @Override
-    public JobId submitFromCatalog(String catalogRestURL, String bucketId, String workflowName,
+    public JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName,
             Map<String, String> variables)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```
Comment (sonarqube-activeeon) [inline rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

File: rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java [hunk 3]
```diff
@@ -197,7 +197,8 @@ JobId submit(URL job, Map<String, String> variables)
      * @throws SubmissionClosedException
      * @throws JobCreationException
      */
-    JobId submitFromCatalog(String catalogRestURL, String bucketId, String workflowName, Map<String, String> variables)
+    JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName,
+            Map<String, String> variables)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException;
```
Comment (sonarqube-activeeon) [inline rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

File: rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java [hunk 4]
```diff
@@ -210,15 +211,15 @@ JobId submitFromCatalog(String catalogRestURL, String bucketId, String workflowN
...
      * @param workflowName
      * @return
      * @throws NotConnectedException
      * @throws PermissionException
      * @throws SubmissionClosedException
      * @throws JobCreationException
      */
-    JobId submitFromCatalog(String catalogRestURL, String bucketId, String workflowName)
+    JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException;
```
Comment (sonarqube-activeeon) [inline rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

File: rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java [hunk 5]
```diff
@@ -734,9 +734,12 @@ public JobId submit(URL job)
     }
 
     @Override
-    public JobId submitFromCatalog(String catalogRestURL, String bucketId, String workflowName)
+    public JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```
Comment (sonarqube-activeeon) [inline rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

File: scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java [hunk 6]
```diff
@@ -321,11 +321,11 @@ public JobId submit(URL job, Map<String, String> variables)
     }
 
     @Override
-    public JobId submitFromCatalog(String catalogRestURL, String bucketId, String workflowName,
+    public JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName,
             Map<String, String> variables)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

File: scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java [hunk 7]
```diff
@@ -292,10 +292,10 @@ public JobId submit(URL job)
     }
 
     @Override
-    public JobId submitFromCatalog(String catalogRestURL, String bucketId, String workflowName)
+    public JobId submitFromCatalog(String catalogRestURL, String bucketName, String workflowName)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java]: MINOR Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 239 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 238 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SchedulerClient.java#L342](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java#L342): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MINOR][MINOR] [ISchedulerClient.java#L99](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L99): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L118](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L118): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L139](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L139): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L159](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L159): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L179](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L179): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L240](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L240): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L251](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L251): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L280](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L280): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L296](https://github.com/ow2-proactive/scheduling/blob/7badac764431ecff5f99093285a9070682041e5b/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L296): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by marcocast** [comment]
retest this please

**Comment by marcocast** [comment]
retest this please

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 239 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 238 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SchedulerClient.java#L342](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java#L342): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MINOR][MINOR] [ISchedulerClient.java#L99](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L99): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L118](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L118): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L139](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L139): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L159](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L159): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L179](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L179): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L240](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L240): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L251](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L251): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L280](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L280): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L296](https://github.com/ow2-proactive/scheduling/blob/66e1b0c4050ce739756b8229755ca5bc42962113/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L296): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 239 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 238 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SchedulerClient.java#L342](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java#L342): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MINOR][MINOR] [ISchedulerClient.java#L100](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L100): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L119](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L119): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L140](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L140): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L160](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L160): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L180](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L180): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L241](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L241): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L252](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L252): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L281](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L281): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L297](https://github.com/ow2-proactive/scheduling/blob/75857a424d0ae57887d61bcedf42ca9d6ee73241/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L297): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 239 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 238 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SchedulerClient.java#L342](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java#L342): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MINOR][MINOR] [ISchedulerClient.java#L100](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L100): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L119](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L119): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L140](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L140): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L160](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L160): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L180](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L180): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L241](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L241): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L252](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L252): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L281](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L281): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. ![MINOR][MINOR] [ISchedulerClient.java#L297](https://github.com/ow2-proactive/scheduling/blob/2c887b5f48ddd2d136ddf48dfd2c0273235b8ded/rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java#L297): Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by ShatalovYaroslav** [comment]
retest this please

**Pull request merged**